server/pkg/asset/preview_type.go

```diff
@@ -53,9 +41,9 @@ func PreviewTypeFromRef(p *string) *PreviewType {
 
 func PreviewTypeFromContentType(c string) *PreviewType {
```

Comment:
I don't think it's unreasonable that everything should be geo except images; if it's not determined to be geo, it should be unknown. identification by extension rather than content-type might be better.

Reply:
Agree!

Reply:
Done!